src/Components/Footer.js

```diff
@@ -1,13 +1,143 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import styled from "styled-components";
 
-class componentName extends Component {
+class Footer extends Component {
   render() {
     return (
-      <div>
-        
+      <div className="Footer">
+        <FooterContainer>
+          <FooterBox>
+            <Top>
+              <Logo></Logo>
+              <TopInfoBox>
+                <li>블로그</li>
+                <li>이용약관</li>
+                <li>개인정보처리방침</li>
+                <li>운영정책</li>
+                <li>고객 문의</li>
+              </TopInfoBox>
+            </Top>
+            <Bottom>
+              <Info>
+                <li>상호명: 주식회사 앤스페이스</li>
+                <li>대표: 정수현</li>
+                <li>사업자등록번호: 230-81-03117</li>
+                <li>통신판매업신고번호: 2018-서울강남-04590</li>
+                <li>사업자등록정보 > </li>
+                <li>영업소재지: 서울특별시 강남구 삼성로 414</li>
+                <li>이메일: office@spacecloud.kr</li>
+                <li>대표전화: 1599-4312(평일 오후 2시 ~ 오후 6시)</li>
+                <li>온라인 1:1문의 바로가기(평일 오전 10시 ~ 오후6시)</li>
+                <li>
+                  *공간에 대한 문의사항은 해당 공간 호스트에서 직접
+                  문의해주세요.
+                </li>
```

Comment:
여기서도 map을 연습해볼 수 있어요!
아래 ```const list```는 컴포넌트 밖에서 정의해주세요!
이 부분 이해 안되면 바로 말씀해주세요 :)
```suggestion

const list = ["상호명: 주식회사 앤스페이스", "대표: 정수현" , ... ]
               {list.map(el => {
                   return (
                       <li>{el}</li>
                   )
               }}
```

Reply:
이해했습니다!!😁